app/home/dashboard.njk

```diff
@@ -35,4 +35,4 @@
       {% endif %}
     {% endfor %}
   </div>
-{% endblock %}
+{% endblock %}
```

Comment:
This looks like an accidental change?

We should keep the newline at the end of the file in these instances too. Maybe look at installing an [editorconfig extension](output = presenter(mockMoves)(mockMoves)) and it should take care of this for you.

Reply:
Good catch.

Editorconfig is installed and running - don't know why it didn't pick up on this first time around, when I'm editing it now, I can't make the newline dissapear. (Will remove this from the PR anyway)

Reply:
what on earth does `[editorconfig extension](output = presenter(mockMoves)(mockMoves))` mean?

Reply:
I assumed the presenter was an accidental `paste` from another comment